pages/posts.js

```diff
@@ -25,14 +23,18 @@ export async function getStaticProps() {
 export default function Posts({ fallback }) {
 	const [isFormActive, setIsFormActive] = useState(false);
 
+	function setFormActiveFalse() {
+		setIsFormActive(false);
+	}
+
 	return (
 		<main>
 			<SWRConfig value={{ fetcher: swrFetcher, fallback }}>
 				<ButtonAdd type="button" onClick={() => setIsFormActive(prevCheck => !prevCheck)}>
 					<Icon variant="plus" />
 				</ButtonAdd>
 				<FormWrapper isFormActive={isFormActive}>
-					<Form onSetIsFormActive={setIsFormActive} />
+					<Form onSetFormActiveFalse={() => setFormActiveFalse()} />
```

Comment:
```suggestion
					<Form onSetFormActiveFalse={setFormActiveFalse} />
```
This works as well and is a bit shorter. But your solution is nice too.